migrations/versions/5cb310101a1f_.py

```diff
@@ -48,7 +48,7 @@ def upgrade():
                     continue
 
                 # Create the sequence with the correct next_id!
-                current_id = session.query(func.max(tbl.c.id)).one()[0] or 1
+                current_id = session.query(func.max(tbl.c.id)).one()[0] or 0
```

Comment:
Are there any databases that might start with a 1?
So we saw this problem with postgres. Would the 0 pose problems with other DBs?

Reply:
There is a `+ 1` when the next id counter gets set:
https://github.com/privacyidea/privacyidea/blob/83c3c6d20e1a878c9ca9e4c738acab1064dc86e7/migrations/versions/5cb310101a1f_.py#L54